V/viterbi/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libviterbi"
```

Comment:
This name ***must*** match the directory where the file is in
```suggestion
name = "viterbi"
```